package.json

```diff
@@ -4,7 +4,6 @@
   "description": "The build toolchain against JavaScript Fatigue",
   "main": "./src/shared/index.js",
   "scripts": {
-    "test": "yarn run lint",
```

Comment:
Yes, but it will be something like
```json
"test": "yarn run lint && yarn jest",
```
So no need to remove it, especially since it's use by circleci :)